### DISCUSSION TITLE
v3.2.36

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#11551] [General]
## What's Changed
* Fixed: Addressed $loop->iteration placement outside loops and correct… by @misaf in https://github.com/filamentphp/filament/pull/11433
* Fix no sorting in ColumnGroup by @pschroee in https://github.com/filamentphp/filament/pull/11463
* Fix `cache-components` command on windows by @saade in https://github.com/filamentphp/filament/pull/11516
* Update .stubs.php by @3adeling in https://github.com/filamentphp/filament/pull/11526
* Fix: default theme mode not respected for pages before login by @wychoong in https://github.com/filamentphp/filament/pull/11527
* Fix form docs custom layout link by @rodrigo-fantuci in https://github.com/filamentphp/filament/pull/11539
* Database transactions by @danharrin in https://github.com/filamentphp/filament/pull/11544
* chore: Laravel Number:: formatting by @danharrin in https://github.com/filamentphp/filament/pull/11545
* Add pivot attributes to Select and CheckboxList components by @saade in https://github.com/filamentphp/filament/pull/11432
* Fix HasOne in repeater by @nettsite in https://github.com/filamentphp/filament/pull/11316
* Use new `hasActualContent` method on slots by @comes in https://github.com/filamentphp/filament/pull/11410
* fix: Nested Spatie relationships by @danharrin in https://github.com/filamentphp/filament/pull/11548
* Ability to add Helper Text to Import Column by @ahinkle in https://github.com/filamentphp/filament/pull/11438
* modifiable select alias for Average, Count and Sum Summerizers by @naseef in https://github.com/filamentphp/filament/pull/11444
* Feature: XLSX format header row styling by @shepard153 in https://github.com/filamentphp/filament/pull/11448
* Group download middlewares within a filament.actions middleware group by @maartenpaauw in https://github.com/filamentphp/filament/pull/11513
* fix: update pgsql generate_search_column_expression by @mgkimsal in https://github.com/filamentphp/filament/pull/11532
* feat: docs Select@createOptionUsing section (03-select.md) by @Carnicero90 in https://github.com/filamentphp/filament/pull/11535
* Fixes the 'guessBelongsToRelationshipTitleColumnName' primary key logic by @IDK-WHO-AM-I in https://github.com/filamentphp/filament/pull/11549
* English grammar corrections by @danswiser in https://github.com/filamentphp/filament/pull/11540

## New Contributors
* @pschroee made their first contribution in https://github.com/filamentphp/filament/pull/11463
* @3adeling made their first contribution in https://github.com/filamentphp/filament/pull/11526
* @rodrigo-fantuci made their first contribution in https://github.com/filamentphp/filament/pull/11539
* @comes made their first contribution in https://github.com/filamentphp/filament/pull/11410
* @ahinkle made their first contribution in https://github.com/filamentphp/filament/pull/11438
* @naseef made their first contribution in https://github.com/filamentphp/filament/pull/11444
* @mgkimsal made their first contribution in https://github.com/filamentphp/filament/pull/11532
* @Carnicero90 made their first contribution in https://github.com/filamentphp/filament/pull/11535
* @IDK-WHO-AM-I made their first contribution in https://github.com/filamentphp/filament/pull/11549
* @danswiser made their first contribution in https://github.com/filamentphp/filament/pull/11540

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.2.35...v3.2.36

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.2.36'>v3.2.36</a>.</em>

**Comment by iyashpal** [comment]
@danharrin Seems the import functionality is broken in this release.
<img width="1117" alt="Screenshot 2024-02-24 at 9 38 52 AM" src="https://github.com/filamentphp/filament/assets/19922707/f22af255-85e6-4751-adda-577697d2313b">
<img width="1117" alt="Screenshot 2024-02-24 at 9 46 06 AM" src="https://github.com/filamentphp/filament/assets/19922707/2ae1e00e-44d3-4d3d-b7cd-af4bc129266a">

**Reply by danharrin** [reply]
https://github.com/filamentphp/filament/releases/tag/v3.2.37
